### DISCUSSION TITLE
Actividad 4 - Grupo 9

### DISCUSSION THREAD

**Discussion by tespinosa9** [frlp-utn-ingsoft/catedra#165] [Show and tell]
https://github.com/EmilianoColavita/vetsoft

**Comment by ivanduva** [comment]
Buenas! Les puedo pedir el link a donde lo deployaron?

**Reply by tespinosa9** [reply]
Hola Iván! Te dejo el enlace a [Render](https://vetsoft-g9.onrender.com)

**Comment by RodrigoJacznik** [comment]
Buenas. Les dejo el feedback de la actividad 3 y 4.

Actividad 3:

Me gusto que hayan elegido un formato para los mensajes de los commits y se hayan comprometido en seguirlo.
Tambien me gusto que hayan armado los fixtures para cargar datos en la db
Los nombres de los branch, el titulo de los PRs y su descripción podrían estar mejor. Son muy pocos descriptivos. Para saber de que se trata un PR tengo que ir a ver el tp.

La actividad esta entregada fuera de termino y algunos PRs necesitan mas trabajo.
https://github.com/EmilianoColavita/vetsoft/pull/6: Hace varias funcionalidades en el mismo PR. No tiene test e2e
https://github.com/EmilianoColavita/vetsoft/pull/10: No permite guardar una mascota con fecha de nacimiento en el futuro pero no muestra ningun error. Seria interesante que lo muestre. Los test e2e fallan.
https://github.com/EmilianoColavita/vetsoft/pull/12: No tiene test de integración ni e2e 

Al final de la actividad (commit fdf03d1), varios test e2e fallan.

Actividad 4.

Encontre varios errores en esta actividad

- No hicieron CI/CD antes de empezar la actividad.
- El CI no corre los tests ni el coverage.
- Tomas y Nicolas no hicieron tareas del punto 3.
- Se repite el problema con los PRs (malos nombres y descripciones)

Les pido por favor que reelean la actividad y corrigan lo siguiente.
Antes de empezar agreguen al CI las tareas que corren los test unitarios, integracion y e2e. Ademas que controle el coverage.

@tespinosa9 y Nicolas, tienen que hacer una de las tareas del punto 3 (no se olviden los test)
Nicolas, agrega los test e2e faltantes del PR https://github.com/EmilianoColavita/vetsoft/pull/6
Ian. Hace que muestre el error al querer guardar una mascota con fecha de nacimiento en el futuro. Hace los test
Tomas. Agrega los test de integración y de e2e del PR https://github.com/EmilianoColavita/vetsoft/pull/12
Emiliano. Hace una de las tarea del punto 3 (que ya no este realizada). No te olvides de escribir los test

La actividad tiene que cumplir la premisa del coverage

Intenten mejorar el nombre de los PRs y su descripcion.

Tienen tiempo hasta el Lunes 8/7.
